pypeman/plugins/remoteadmin/views.py

```diff
@@ -26,11 +26,15 @@ async def list_channels(request, ws=None):
 
     chans = []
     for chan in channels.all_channels:
-        if not chan.parent:
-            chan_dict = chan.to_dict()
+        chan_dict = chan.to_dict()
+        if chan_dict["has_message_store"]:
             chan_dict['subchannels'] = chan.subchannels()
-
             chans.append(chan_dict)
+            # if not chan.parent:
```

Comment:
what is the purpose of the commented lines?

perhaps add a 'header line'

like

 # to be implemented soon

 # this is how you do ...

 # any other description

Reply:
no purpose, deleted